### ISSUE TITLE
bash syntax issue in `version.sh`

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#1030]
Hi @ericawood I think I must have messed up version.sh:
```
+ echo '================= starting version.sh =================='
================= starting version.sh ==================
+ date
Thu Aug 20 02:17:19 UTC 2020
++ dirname /home/ubuntu/kg2-code/version.sh
+ CONFIG_DIR=/home/ubuntu/kg2-code
+ source /home/ubuntu/kg2-code/master-config.shinc
++ '[' -z ']'
++ test_suffix=
++ BUILD_DIR=/home/ubuntu/kg2-build
++ VENV_DIR=/home/ubuntu/kg2-venv
++ CODE_DIR=/home/ubuntu/kg2-code
++ umls_dir=/home/ubuntu/kg2-build/umls
++ umls_dest_dir=/home/ubuntu/kg2-build/umls/META
++ s3_region=us-west-2
++ s3_bucket=rtx-kg2
++ s3_bucket_public=rtx-kg2-public
++ s3_bucket_versioned=rtx-kg2-versioned
++ s3_cp_cmd='aws s3 cp --no-progress --region us-west-2'
++ mysql_conf=/home/ubuntu/kg2-build/mysql-config.conf
++ curl_get='curl -s -L'
++ curies_to_categories_file=/home/ubuntu/kg2-code/curies-to-categories.yaml
++ curies_to_urls_file=/home/ubuntu/kg2-code/curies-to-urls-map.yaml
++ predicate_mapping_file=/home/ubuntu/kg2-code/predicate-remap.yaml
++ ont_load_inventory_file=/home/ubuntu/kg2-code/ont-load-inventory.yaml
++ umls2rdf_config_master=/home/ubuntu/kg2-code/umls2rdf-umls.conf
++ rtx_config_file=RTXConfiguration-config.json
+ trigger_file_is_major_release=/home/ubuntu/kg2-build/major-release
+ local_version_filename=/home/ubuntu/kg2-build/kg2-version.txt
+ build_flag=
+ s3_version_filename=kg2-version.txt
+ aws s3 cp --no-progress --region us-west-2 s3://rtx-kg2-public/kg2-version.txt /home/ubuntu/kg2-build/kg2-version.txt
download: s3://rtx-kg2-public/kg2-version.txt to ./kg2-version.txt
+ '[' '' == test
/home/ubuntu/kg2-code/version.sh: line 29: [: missing `]'
+ '' == alltest ']'
/home/ubuntu/kg2-code/version.sh: line 29: : command not found
+ '[' -e /home/ubuntu/kg2-build/major-release ']'
+ increment_flag=--increment_minor
+ '[' --increment_minor '!=' '' ']'
+ /home/ubuntu/kg2-venv/bin/python3 /home/ubuntu/kg2-code/update_version.py --increment_minor /home/ubuntu/kg2-build/kg2-version.txt
+ aws s3 cp --no-progress --region us-west-2 /home/ubuntu/kg2-build/kg2-version.txt s3://rtx-kg2-public/kg2-version.txt
upload: ./kg2-version.txt to s3://rtx-kg2-public/kg2-version.txt
+ '[' -f /home/ubuntu/kg2-build/major-release ']'
+ date
Thu Aug 20 02:17:22 UTC 2020
+ echo '================= finishing version.sh =================='
```

**Comment by saramsey** [comment]
never mind, I fixed it.

**Comment by saramsey** [comment]
I have fixed the issue and resumed the v2.3.0 build on `kg2dev`.
